source/api/accounts-multi.md

```diff
@@ -50,7 +50,8 @@ These functions return an object with a single method, `stop`.  Calling
 
 On the server, the callbacks get a single argument, the same attempt info
 object as [`validateLoginAttempt`](#accounts_validateloginattempt). On the
-client, no arguments are passed.
+client, an object is supplied containing only the error from the failed
```

Comment:
I might put "error" in backticks to make it known that it's the property name in the object.  Additionally, since it's not clear whether the error is a `String` or what (it's a JavaScript `Error` object, I believe?), maybe something along the lines of:

```
client, an object containing a single `error` property set to the `Error`-object
which was received from the failed login attempt.
```

?